qt/lib/include/verovio-qt/veroviodoc.h

```diff
@@ -0,0 +1,208 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        veroviodoc.h
+// Author:      Jonathan Schluessler
+// Created:     2017
+// Copyright (c) Authors and others. All rights reserved.
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef __VEROVIODOC_H__
+#define __VEROVIODOC_H__
+
+//----------------------------------------------------------------------------
+
+#include <QObject>
+#include <QVector>
+
+//----------------------------------------------------------------------------
+
+#include "toolkit.h"
+
+namespace vrv {
+
+//----------------------------------------------------------------------------
+// VerovioDoc
+//----------------------------------------------------------------------------
+
+/**
+ * This class represents one Verovio document.
+ */
+class VerovioDoc : public QObject {
```

Comment:
Since this class is a wrapper around the Verovio toolkit I would not call it VerovioDoc but instead something like QtVrvToolkit

Reply:
Yes, that makes sense. Instead of using a namespace in the class name (QtVrvToolkit) I've decided to put all classes of verovio-qt in its own namespace `vrv_qt`. I first thought about using a nested namespace (`vrv::qt`) but that leads to indenting everying inside of that namespace by the used clang-format.